dts/bindings/ethernet/ethernet.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Add license & copyright header info, see dts/bindings/gpio/gpio-leds.yaml for an example